code/_globalvars/lists/maintenance_loot.dm

```diff
@@ -213,6 +213,14 @@ GLOBAL_LIST_INIT(uncommon_loot, list(//uncommon: useful items
 	list(//food
 		/obj/item/reagent_containers/food/snacks/canned/peaches/maint = 1,
 		/obj/item/storage/box/donkpockets = 1,
+		list(//Donk varities
+			/obj/item/storage/box/donkpockets/donkpocketspicy = 1,
+			/obj/item/storage/box/donkpockets/donkpocketteriyaki = 1,
+			/obj/item/storage/box/donkpockets/donkpocketpizza = 1,
+			/obj/item/storage/box/donkpockets/donkpocketgondola= 1,
```

Comment:
This makes it possible for gondola pockets to spawn in maint. If that's not intentional you should probably 86 it.

Reply:
Good find! Its been removed.